src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -0,0 +1,13 @@
+import { T } from '../../../translate-marker';
+
+export default {
+    step1_label: T('SSH Keypair/Connection'),
+
+    name_placeholder: T('Name'),
+    name_tooltip: T(''),
```

Comment:
```suggestion
    name_tooltip: T('Replication Task Name'),
```

Reply:
@a-stjohn It is not only for Replication task. The name also use for new created ssh keypair/ ssh connection/period snapshot task.